### DISCUSSION TITLE
Uploaded Jupyter notebook does not open on GitHub

### DISCUSSION THREAD

**Discussion by melisasaygin** [community/community#124651] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I uploaded many Jupyter notebooks to a repository about 24 hours ago and they were all visible in the preview without any issues. Now, only time passes, I changed nothing, and when I click, the Jupyter notebook preview does not open. It says: An error occurred Using nbformat v5.10.4 and nbconvert v7.16.1

**Comment by davevad93** [comment]
Hi @melisasaygin ,

As you can see in these discussions, #124617  , #124611 , #124601 , #124597 is happening to all GitHub users today, there are issues with rendering `ipynb` files, I guess they will fix it soon. If you need to view your files for now you can use [nbviewer](https://nbviewer.org/) or  [ReviewNB](https://www.reviewnb.com/?utm_source=reviewnb_blog).

**Reply by melisasaygin** [reply]
Thank you for the heads-up!

**Reply by davevad93** [reply]
You're welcome @melisasaygin , I guess it won't be fixed before tomorrow considering today is Sunday but I hope it won't take too long.

**Reply by qyangcv** [reply]
The bug still exists, and nbviewer truly helps me.

**Reply by Juan-Pablo-17** [reply]
Thank you, I have been trying to upload a notebook from kaggle and it did not load and checking my old repositories I realized that they were not loading. Thanks for the heads-up! 😁

**Reply by 40tude** [reply]
Same issue today May 5 2024. Thanks for sharing information

**Reply by mohammedshaneeb-ai** [reply]
The bug still exists, Today May 20 2024

**Reply by davevad93** [reply]
Update @melisasaygin , the problem has been fixed, it's now possible to view `ipynb` files as before.

**Reply by chriskambimbi** [reply]
i am still unable to see my ipynb 

Invalid Notebook
missing attachment: a04178-c30e-4fd-b2c2-349e130.png
Using nbformat v5.10.4 and nbconvert v7.16.1

**Reply by 223047** [reply]
> i am still unable to see my ipynb
> 
> Invalid Notebook missing attachment: a04178-c30e-4fd-b2c2-349e130.png Using nbformat v5.10.4 and nbconvert v7.16.1

Canhductu121.com

**Reply by cemputer** [reply]
The bug still exists, Today Jun 22 2025

**Comment by XuejiaoBian** [comment]
Same error happened today @melisasaygin 
It seems the newly edited ones would have this problem?
![微信图片编辑_20240519211348](https://github.com/community/community/assets/103819504/32602af1-d4e4-48f1-bafa-c99deef888c3)

**Reply by melisasaygin** [reply]
@XuejiaoBian yes I see the same exact thing. I hope it gets fixed soon

**Reply by SummerShao** [reply]
Wow, same issue

**Reply by souravmalik123** [reply]
Same issue here please any body help me

**Reply by siadis** [reply]
Same issue here. Unfortunately. 
<img width="491" alt="error github preview ipynb file" src="https://github.com/community/community/assets/166292626/d950aa1e-5e0a-4835-ae22-22ead5ca59ef">

**Comment by ProfAllanIFBA** [comment]
From what I saw, you can open it normally through colab. It's just a direct view issue on github.

**Reply by alldominguez** [reply]
It is a rendering problem from GitHub

**Comment by GalkaKG** [comment]
Same error here:
![image](https://github.com/community/community/assets/103485495/144ee3d0-9dc6-4082-9730-d8842aa91f3e)

**Comment by samtaba** [comment]
Happening to me as well.

**Comment by Juan-Carlos-D** [comment]
Happening to me as well since yesterday! Hope it gets fixed soon

**Comment by lancy-buding** [comment]
Thank you so much, I just thought maybe it is my laptop question....

**Comment by Muthukamalan** [comment]
Happening to me as well

**Comment by RubhiniAnand** [comment]
same issue

**Comment by rayssathayde** [comment]
Same here, can’t access any Jupyter notebook.

Rayssa Maciel Athayde
Bióloga
Mestre em Biologia Celular
Professora de Ciências e Biologia


Em dom., 19 de mai. de 2024 às 13:56, Rubhini Anand <
***@***.***> escreveu:

**Comment by Filichkin** [comment]
Same issue, I still wait...

**Comment by 0xrushi** [comment]
Happening to me as well

**Comment by sabalunax** [comment]
I'm still waiting

**Comment by elfchildRichter** [comment]
Same issue, I also thought whether I had mistakenly set something... hope it gets fixed soon

**Comment by rabirabii** [comment]
Hope they fix it soon

**Comment by josephassiga** [comment]
Happened to me as well

**Comment by Yaseminh** [comment]
I got this error. how can I solve this?

**Comment by nisargnegi** [comment]
Facing the same issue, lets hope it gets fixed soon

**Comment by AIdjis** [comment]
same error, but The problem remains unresolved.
![Screenshot 2024-05-19 235048](https://github.com/community/community/assets/88404339/69a1fa9d-bb1b-40fe-9c30-dd961c7f8533)

**Comment by tusharx0809** [comment]
Its still not fixed

**Comment by noe-sautel** [comment]
You can copy and paste your link to this viewer in the meantime: https://nbviewer.org/

GitHub will correct this bug for the next few days as it is a global issue.

**Comment by samzong** [comment]
still no working.

**Comment by jawatech** [comment]
same here

**Comment by bhargavpandya1** [comment]
Same here. Not opening any notebooks

**Comment by plutonium-guy** [comment]
same

**Comment by bytexlML** [comment]
same issue here
![image](https://github.com/community/community/assets/166785795/ac59afe5-14cf-4fe0-9366-3db143c7774d)

**Comment by kaushikmastro** [comment]
Same error.

**Comment by bzadrozny-vimn** [comment]
same here

**Comment by yasemindk** [comment]
++++

**Reply by tarun-kumar2** [reply]
same here

**Comment by christoffer-apell** [comment]
Same here.

**Comment by FilippoFlorindiKAI** [comment]
Same here.

**Comment by mzahran001** [comment]
Same here.

**Comment by kalpanasanikommu** [comment]
Same here.

**Comment by GonzaloMartin** [comment]
Same issue for me,  I'm using Collab instead until it get fixed.

Cheers!

**Comment by efrainmpp1** [comment]
Same issue for me too. :(

**Comment by dpaysan** [comment]
and for me as welll

**Comment by Paul-Anderson2** [comment]
Same problem - seems to be across all of Github

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @melisasaygin ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Comment by gorokhovnik-trailmix** [comment]
Same problem: both for old and new notebooks

**Comment by orcunmadran** [comment]
Same problem for Colab notebooks..!

**Comment by rahulbhoyar1995** [comment]
Same problem for me as well.

**Comment by steveplexagon** [comment]
My notebooks work in codespace ( open as jupyter notebook)

**Reply by mughal1999** [reply]
To work with Jupyter notebooks in GitHub Codespaces:

Access Codespaces:

Navigate to your GitHub repository and click "Code" > "Open with Codespaces."
Environment Setup:

Codespaces will set up a cloud-based development environment.
Open Notebook:

Use VS Code in Codespaces to open your Jupyter notebook (.ipynb file).
Use VS Code for Jupyter:

VS Code has built-in Jupyter support for editing, running code, and viewing output.
Save and Share:

Save changes in VS Code; notebooks are automatically saved to your GitHub repository.

**Comment by theothertomelliott** [comment]
Looks like an incident was posted for this and now resolved:
https://www.githubstatus.com/incidents/915nn7rmqth9

Our notebooks are looking good, so can confirm it's fixed for us at least.

**Comment by mph4** [comment]
Hi Everyone - very sorry for the inconvenience here. We've fixed the problem and are also implementing improvements that we've learned from this to help prevent it (and better respond to any future issues) going forward.

**Reply by chriskambimbi** [reply]
not able to save my notebook 
![image](https://github.com/community/community/assets/168649859/762856eb-5ec8-4124-ba2e-aba9619176fd)

**Comment by Juan-Pablo-17** [comment]
Hello, I have a problem when uploading my JupyterNotebook file from Kaggle, when I upload it and do the push, this is what my repository shows
![image](https://github.com/community/community/assets/113142598/54a8d6fd-9a1a-434e-b73b-3476140d44bc)

**Comment by Larsibaer** [comment]
Now it's fixed

**Comment by mughal1999** [comment]
Happening to me as well.

**Comment by 223047** [comment]
> ### Select Topic Area
> Bug
> 
> ### Body
> I uploaded many Jupyter notebooks to a repository about 24 hours ago and they were all visible in the preview without any issues. Now, only time passes, I changed nothing, and when I click, the Jupyter notebook preview does not open. It says: An error occurred Using nbformat v5.10.4 and nbconvert v7.16.1

Hoangvancanh7343

**Comment by 223047** [comment]
Canhductu121.com

**Comment by 223047** [comment]
Canhductu121@gmail.com

**Comment by 223047** [comment]
Hoangvancanh17071980@gmail.com

**Comment by 223047** [comment]
@Hcanh25365

**Comment by werruww** [comment]
![Untitled](https://github.com/user-attachments/assets/a8c4f293-7cc2-4bd2-8f11-32f1deadc382)

**Comment by Shivam6035** [comment]
this issues still exist in 2026

**Reply by yokotsuno12** [reply]
Hello! Are you still having this issue on the 31st? Thanks!

**Comment by Fariela** [comment]
The issue still exist May 30 !!

**Comment by tanvishinde017** [comment]
I’m seeing the same issue. I uploaded several Jupyter notebooks about 24 hours ago, and their previews were working correctly at that time. Since then, I have not modified the notebooks or repository settings, but the notebook preview now fails to load and displays:

> An error occurred
> Using nbformat v5.10.4 and nbconvert v7.16.1

This appears to be a regression or a problem with the notebook rendering service rather than with the notebook files themselves. Could someone confirm whether there is an ongoing issue with notebook preview rendering?

**Reply by yokotsuno12** [reply]
Hello, same issue with preview rendering here. :( 
It's possible to open the notebooks through Colab though.

**Reply by marcocruzchavez** [reply]
I'm having the same problem with the notebooks and getting the same error

**Comment by marcocruzchavez** [comment]
I'm having the same problem with the notebooks and getting the same error

**Comment by Shivam6035** [comment]
can someone suggest any solution

**Comment by Shubh-0077** [comment]
Same here 
<img width="1918" height="860" alt="Screenshot 2026-05-31 130637" src="https://github.com/user-attachments/assets/c8e78b67-557b-4015-84a7-214afee1a7c7" />
